### ISSUE TITLE
KeepKey emits a lot of events

### ISSUE THREAD

**Issue by 0xApotheosis** [shapeshift/hdwallet#505]
When interacting with a KeepKey via `web`, the device is noisy - many events are emitted, likely more than is necessary.

This could also be a `web` issue - TBC.

The screenshot below shows the number of events emitted when connecting a KeepKey and correctly entering a PIN.

<img width="605" alt="Screen Shot 2022-04-07 at 12 55 32 pm" src="https://user-images.githubusercontent.com/97164662/162111497-e1228c51-f2dc-4529-9fb6-51b7d287cad1.png">

**Comment by mrnerdhair** [comment]
Yep. There's an event emitted for each packet send or received to the device, and I don't think any of them are actually used. We really need to clean up the way we use EventEmitters in hdwallet, and concretely specify which actual events are intended to be part of the public interface; as is, I think we have to prove definitively these events aren't being used anywhere to consider removing them a non-breaking change.
